db/seeds/playbook.seeds.rb

```diff
@@ -13,7 +14,8 @@
    description: 'Yeah, I’ve been around the block. A bit of this, a
 bit of that. When I came across the secret underworld of monsters
 and magic… well… it wasn’t so different from the underworld I
-already knew. It was easy to find an angle, just like before.'
+already knew. It was easy to find an angle, just like before.',
+   categories: 'transport'

```

Comment:
Added categories here.  I do feel like they would fit better in the individual playbooks, but I did it here as a work around solution until the individual playbooks are built out.